www/functions_CAM/create_ListSynonyms.R

```diff
@@ -83,4 +83,4 @@ if(length(listSynonyms) == 1) {
 
   return(listSynonyms)
 }
-}
\ No newline at end of file
+}
```

Comment:
Sorry, just noticed the missing blank line at the end. That's not intentional, don't know why I "lost" the line. So don't follow me on this one ;-)

Reply:
gefixxt danke :)